src/services/blocks/BlocksService.ts

```diff
@@ -338,13 +341,14 @@ export class BlocksService extends AbstractService {
 		block: Block
 	) {
 		const perByte = api.consts.transactionPayment?.transactionByteFee;
-		const extrinsicBaseWeight = api.consts.system
-			?.extrinsicBaseWeight as AbstractInt;
+		const extrinsicBaseWeightExists =
+			api.consts.system.extrinsicBaseWeight ||
+			api.consts.system.blockWeights.perClass.normal.baseExtrinsic;
```

Comment:
We are making an assumption here that if a runtime has one of these in the metadata currently, all past runtimes will have it. Since we only support fee calcs on a limited number of chains I think this is ok for now